python/py-Pillow/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-Pillow
-version             9.5.0
+version             10.0.1
```

Comment:
version 10.0.x does not support PY37 anymore, so you will need to pin that subport to the latest supported version

Reply:
I've added a commit.